lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
       margin: 0;
       padding: 0;
       list-style: none;
 
       .courses-listing-item {
-        width: flex-grid(4);
-        @include margin-right(flex-gutter());
-        @include float(left);
+        height: 360px;
+
+        @include media($mobile) {
+          @include span-columns(4);
+          margin: 15px 0px;
```

Comment:
Reviewing again now - are these left/right margins needed assuming we're using Neat's ``$gutter`` (http://thoughtbot.github.io/neat-docs/latest/#gutter)? This should give any elements assigned a grid width proportional left/right margins. I'd recommend relying on those rather than setting our own if possible.